frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ManageServingRuntimeModal.tsx

```diff
@@ -58,15 +57,17 @@ const ManageServingRuntimeModal: React.FC<ManageServingRuntimeModalProps> = ({
   const [actionInProgress, setActionInProgress] = React.useState(false);
   const [error, setError] = React.useState<Error | undefined>();
 
-  const { currentProject } = React.useContext(ProjectDetailsContext);
-
-  const { dashboardNamespace } = useDashboardNamespace();
+  const {
+    currentProject,
+    servingRuntimesConfig: { servingRuntimesConfig },
+  } = React.useContext(ProjectDetailsContext);
```

Comment:
@andrewballantyne did't get the full extend of gpu's changes, and now I'm seeing that we had redundant code. I removed my hook so the Nit you had is gone. To answer your question, that was done in order to refresh the results on the modal appearing, I'm gonna try to add the refresh here in case the config changes.

Reply:
Ok, changed using the loading state